Library/Homebrew/extend/os/mac/dev-cmd/bottle.rb

```diff
@@ -0,0 +1,31 @@
+# typed: false
+# frozen_string_literal: true
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  def setup_tar_and_args!(args, mtime)
+    generic_setup_tar_and_args!(args, mtime)
+
+    # Use gnu-tar on macOS as it can be set up for reproducibility better than libarchive.
+    begin
+      gnu_tar = Formula["gnu-tar"]
+    rescue FormulaUnavailableError
+      return default_tar_args
+    end
+
+    ensure_formula_installed!(gnu_tar, reason: "bottling")
+
+    # Ensure tar is set up for reproducibility.
+    # https://reproducible-builds.org/docs/archives/
+    gnutar_args = [
```

Comment:
I think it'd be good to share this method too given it's the same? Could add a new `gnutar_args` method to `bottle.rb`